client/database.go

```diff
@@ -1,96 +0,0 @@
-package client
-
-import (
-	"database/sql"
-	"fmt"
-
-	_ "github.com/go-sql-driver/mysql"
-)
-
-const numTables int = 5
-
-func createTable(numTable int, db *sql.DB) error {
-	stmt := fmt.Sprintf(`CREATE TABLE IF NOT EXISTS User%02d (
-		id int NOT NULL AUTO_INCREMENT,
```

Comment:
ah I see you put in code... another thought is to take advantage of docker which you could configure to spin up a mysql instance and then populate it every time... e.g. https://github.com/jimmyjames85/bpmonitor